rbc/merkle_tree.go

```diff
@@ -0,0 +1,79 @@
+package rbc
+
+import (
+	"bytes"
+	"crypto/sha256"
+
+	"github.com/cbergoon/merkletree"
+)
+
+type MerkleData = merkletree.Content
+
+type RootPath [][]byte
+type Data []byte
+
+func (data Data) CalculateHash() ([]byte, error) {
+	h := sha256.New()
+	if _, err := h.Write([]byte(data)); err != nil {
+		return nil, err
+	}
+
+	return h.Sum(nil), nil
+}
+
+func (data Data) Equals(other merkletree.Content) (bool, error) {
+	return bytes.Equal(data, other.(Data)), nil
+}
+
+func NewMerkleTree(datas [][]byte) (*merkletree.MerkleTree, error) {
+	var merkleDatas []MerkleData
+
+	for _, data := range datas {
+		merkleDatas = append(merkleDatas, Data(data))
+	}
+
+	tree, err := merkletree.NewTree(merkleDatas)
+	if err != nil {
+		return nil, err
+	}
+
+	return tree, nil
```

Comment:
plz wrapping this tree, so that we do not expose `merkletree.MerkleTree` to exteranl